sql/src/lib.rs

```diff
@@ -160,7 +160,14 @@ impl QueryBuilder for SQLiteQueryBuilder {
             &Ref(entid) => self.push_sql(entid.to_string().as_str()),
             &Boolean(v) => self.push_sql(if v { "1" } else { "0" }),
             &Long(v) => self.push_sql(v.to_string().as_str()),
-            &Double(OrderedFloat(v)) => self.push_sql(v.to_string().as_str()),
+            &Double(OrderedFloat(v)) => {
+                // Rust's floats print without a trailing '.' in some cases.
```

Comment:
I did some experimentation in a Rust playground for this.

Using `{:.1}` doesn't behave as you'd expect:

https://is.gd/8QaoD5

```
fn main() {
    let i: f64 = 123.0f64;                     // Integer.
    let v: f64 = 9999.00123f64;
    let x: f64 = 9.99900123e3f64;
    let z: f64 = 1.000123f64;
    println!(":: {:e} vs {:.1} vs {}", i, i, i);
    println!(":: {:e} vs {:.1} vs {}", x, x, x);
    println!(":: {:e} vs {:.1} vs {}", v, v, v);
    println!(":: {:e} vs {:.1} vs {}", z, z, z);
        
    let s = "9.99900123e3";
    let parsed = s.parse::<f64>().unwrap();
    println!("Parsed to {:e}, {:.1}, {}", parsed, parsed, parsed);
    println!("Same: {}, {}", parsed == v, parsed == x);
}
```

=>

```
:: 1.23e2 vs 123.0 vs 123
:: 9.99900123e3 vs 9999.0 vs 9999.00123
:: 9.99900123e3 vs 9999.0 vs 9999.00123
:: 1.000123e0 vs 1.0 vs 1.000123
Parsed to 9.99900123e3, 9999.0, 9999.00123
Same: true, true
```

As you can see, the only solution that round-trips, never prints as an integer, and always preserves precision is `{:e}`.